default.nix

```diff
@@ -36,6 +36,8 @@ in haskellPackages.developPackage {
       [
         pkgs.nix
         haskellPackages.hpack
+        haskellPackages.http-client
```

Comment:
This shouldn't be necessary, `developPackage` should pick it up for you

Reply:
Right, we only need things here which are not referenced in `package.yaml`.